apps/organisations/forms.py

```diff
@@ -35,22 +35,46 @@
 ]
 
 SOCIAL_MEDIA_SIZES = {
-    1: {'title_max_length': 128,
-        'description_max_length': 256,
+    1: {'title_max_length': 30,
+        'title_size': 72,
+        'description_max_length': 45,
+        'description_size': 48,
         'img_min_width': 1080,
-        'img_min_height': 1080},
-    2: {'title_max_length': 128,
-        'description_max_length': 256,
+        'img_min_height': 760,
+        'aplus_logo_width': 228,
+        'aplus_logo_height': 56,
+        'overall_height': 1080
+        },
+    2: {'title_max_length': 30,
+        'title_size': 72,
+        'description_max_length': 45,
+        'description_size': 48,
         'img_min_width': 1080,
-        'img_min_height': 1920},
-    3: {'title_max_length': 128,
-        'description_max_length': 256,
-        'img_min_width': 1104,
-        'img_min_height': 736},
-    4: {'title_max_length': 128,
-        'description_max_length': 256,
+        'img_min_height': 1278,
+        'aplus_logo_width': 360,
+        'aplus_logo_height': 88,
+        'overall_height': 1920
+        },
```

Comment:
NOTE: update styling of story to match new design and change name to 'Instagram story'